src/serializers/ob_type.rs

```diff
@@ -23,6 +23,7 @@ pub struct ObTypeLookup {
     list: usize,
     dict: usize,
     // other numeric types
+    decimal_object: PyObject,
     decimal: usize,
```

Comment:
`decimal_object.as_ptr() as usize` should be a zero-cost cast so I don't believe you need to store the `usize` forms. (If you disagree, can benchmark)

Reply:
is that true for all the `PyObject`s here?

Reply:
It should be, yes. (`PyObject` is internally just `NonNull<ffi::PyObject>` so the `.as_ptr()` is just casting itself to `*mut ffi::PyObject`.)

Reply:
done.